src/V3Broken.cpp

```diff
@@ -28,220 +28,158 @@
 #include "V3Global.h"
 #include "V3Broken.h"
 #include "V3Ast.h"
+#include "V3Error.h"
 
 // This visitor does not edit nodes, and is called at error-exit, so should use constant iterators
 #include "V3AstConstOnly.h"
 
-#include <algorithm>
 #include <unordered_map>
 #include <unordered_set>
 
 //######################################################################
+// Generation counter for AstNode::m_brokenState
 
-class BrokenTable VL_NOT_FINAL : public AstNVisitor {
-    // Table of brokenExists node pointers
+static class BrokenCntGlobal {
+    // Value 0 is invalid, as the AstNode constructor uses that to initialize m_brokenState
+    uint8_t m_count = 1;
+
+public:
+    uint8_t get() {
+        UASSERT(1 <= m_count && m_count <= 127, "Invalid generation number");
```

Comment:
Hmm, thought commented earlier but lost it.

Please use a constexpr with 127 and use here and in the below 127.  Add a note that this corresponds to 7 bits in m_brokenState, and vice-versa, thanks.